packages/mdc-grid-list/_mixins.scss

```diff
@@ -0,0 +1,249 @@
+@import "./variables";
+
+@mixin mdc-grid-list-core-styles($query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  // postcss-bem-linter: define grid-list
+  @at-root {
+    @include mdc-grid-list-tile-aspect(1, $query: $query);
+    @include mdc-grid-list-tile-gutter(4px, $query: $query);
+  }
+
+  .mdc-grid-list__tiles {
+    @include mdc-feature-targets($feat-structure) {
+      display: flex;
+      flex-flow: row wrap;
+      padding: 0;
+    }
+  }
+
+  .mdc-grid-list--tile-gutter-1 {
+    @include mdc-grid-list-tile-gutter(1px, $query: $query);
+  }
+
+  .mdc-grid-list--tile-aspect-16x9 {
+    @include mdc-grid-list-tile-aspect(16 / 9, $query: $query);
+  }
+
+  .mdc-grid-list--tile-aspect-3x2 {
+    @include mdc-grid-list-tile-aspect(3 / 2, $query: $query);
+  }
+
+  .mdc-grid-list--tile-aspect-2x3 {
+    @include mdc-grid-list-tile-aspect(2 / 3, $query: $query);
+  }
+
+  .mdc-grid-list--tile-aspect-4x3 {
+    @include mdc-grid-list-tile-aspect(4 / 3, $query: $query);
+  }
+
+  .mdc-grid-list--tile-aspect-3x4 {
+    @include mdc-grid-list-tile-aspect(3 / 4, $query: $query);
+  }
+  // postcss-bem-linter: end
+
+  // postcss-bem-linter: define grid-tile
+  .mdc-grid-tile {
+    @include mdc-feature-targets($feat-structure) {
+      display: block;
+      position: relative;
+      /* @alternate */
+      width: $mdc-grid-list-tile-width;
+      width: var(--mdc-grid-list-tile-width, $mdc-grid-list-tile-width);
+    }
+  }
+
+  .mdc-grid-tile__primary {
+    @include mdc-feature-targets($feat-color) {
+      @include mdc-theme-prop(background-color, background);
+      @include mdc-theme-prop(color, text-primary-on-background);
+    }
+
+    @include mdc-feature-targets($feat-structure) {
+      position: relative;
+      height: 0;
+    }
+  }
+
+  .mdc-grid-tile__primary-content {
+    @include mdc-feature-targets($feat-structure) {
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      background-repeat: no-repeat;
+      background-position: center;
+      background-size: cover;
+    }
+  }
+
+  .mdc-grid-tile__secondary {
+    @include mdc-feature-targets($feat-color) {
+      @include mdc-theme-prop(background-color, primary);
+      @include mdc-theme-prop(color, on-primary);
+    }
+
+    @include mdc-feature-targets($feat-structure) {
+      position: absolute;
+      bottom: 0;
+      box-sizing: border-box;
+      height: $mdc-grid-list-oneline-cap-secondary-height;
+      padding: $mdc-grid-list-tile-secondary-padding;
+    }
+  }
+
+  .mdc-grid-tile__title {
+    @include mdc-typography-overflow-ellipsis($query: $query);
+
+    @include mdc-feature-targets($feat-structure) {
+      display: block;
+      margin: 0;
+      padding: 0;
+      font-size: 1rem;
+      font-weight: #{map-get($mdc-typography-font-weight-values, medium)};
```

Comment:
should this one be under typography?

Reply:
Whoops, looks like it. Thanks for catching that.